archive/v/verilog/README.md

```diff
@@ -5,6 +5,7 @@ Welcome to Sample Programs in Verilog!
 ## Sample Programs
 
 - [Hello World in Verilog](https://github.com/jrg94/sample-programs/issues/97)
+- [Fibonacci Sequences in Verilog](https://github.com/TheSupremePatel/sample-programs/blob/verilog/fibonacci/archive/v/verilog/fibonacci_behavioural.v)
```

Comment:
We typically try to link either an article explaining the snippet or an issue which calls to the need for an article here.

Reply:
I'll try to make changes to the current PR. Thanks for the response

Reply:
I had initially tried to use command line but I did not find any ideal solution for that.